Mathlib/Logic/Nontrivial/Defs.lean

```diff
@@ -44,6 +41,7 @@ protected theorem Decidable.exists_ne [Nontrivial α] [DecidableEq α] (x : α)
     exact ⟨y', h.symm⟩
   · exact ⟨y, Ne.symm hx⟩
 
+open Classical in
 theorem exists_ne [Nontrivial α] (x : α) : ∃ y, y ≠ x := Decidable.exists_ne x
```

Comment:
This is the only change I'm not fully sure of.

Reply:
Why?

Reply:
I'm unsure in the sense of "should I add a suitable decidability assumption instead?" --- though come to think of it, the answer is "no".